frontend/public/components/_monitoring.scss

```diff
@@ -30,6 +30,10 @@ $monitoring-line-height: 18px;
   text-overflow: ellipsis;
 }
 
+.monitoring-no-padding-left {
```

Comment:
This is not a great name.  Maybe `monitoring-reg-ex` or something that speaks to the functionality?

Reply:
Changed to `monitoring-silence-reg-ex`